README.adoc

```diff
@@ -1,3 +1,25 @@
+= C++ example project scanned on SonarCloud using Bitbucket Cloud pipelines
+// TODO: add SC status
+// TODO: add link to SC
+
+To analyze a C/C++/Objective-C project on Bitbucket pipelines with SonarCloud, follow the steps described here: https://sonarcloud.io/documentation/getting-started/bitbucket-cloud/[Getting started with Bitbucket Cloud].
+
+To sum up, you need to:
+
+. Enable pipelines in your Bitbucket Cloud repository
```

Comment:
Perhaps, keep the step about filling sonar-project.properties, as in other examples.